src/helpers/check-merge-safety.ts

```diff
@@ -96,12 +100,21 @@ const getMergeSafetyMessage = async (
     basehead: `${baseOwner}:${default_branch}...${username}:${ref}`
   });
   const changedFileNames = changedFiles?.map(file => file.filename);
-  const projectDirectories = paths?.split(/[\n,]/);
-  const isUnsafeToMerge = projectDirectories?.some(
+  const changedProjectDirectories = paths?.split(/[\n,]/);
```

Comment:
are we expecting different usage from `paths` now?

Reply:
No, this is just a refactor that allows us to log out the specific projects or files that caused the failure

Reply:
`paths` should actually be the complete list of projects in the repo, not just the changed packages. While the list of changed packages would work for the PR workflow, it would not work for the push workflow since that workflow doesn't know the changed packages of each PR

Reply:
so if `paths` is a complete list of projects, why is the list called `changedProjectDirectories`?

Reply:
That's a really good question, it shouldn't be. 🤦